nodejs/week2/warmUpB/index2.js

```diff
@@ -0,0 +1,32 @@
+const express = require("express");
+const app = express();
+
+app.get("/numbers/add", (req, res) => {
+  const first = parseInt(req.query.first);
+  const second = parseInt(req.query.second);
+  let sum = parseInt(first + second);
+  if (!isNaN(first) && first > 0 && !isNaN(second) && second > 0) {
```

Comment:
usually it's good to exit early instead. So:
`if (isNan(first) || isNan(second)) {
   console.log(`first and second argument need to be a number`);
   return;
}`
after this point, you can do all the logic assuming your inputs are correct.